packages/react-devtools-extensions/iframe/iframe-other-domain.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <iframe src="http://127.0.0.1:8080/packages/react-devtools-extensions/iframe/main.html" />
```

Comment:
These files don't belong in this package. Packages are for source code. There's a separate folder for fixtures:

https://github.com/facebook/react/tree/master/fixtures/devtools